src/components/buttons/LinkButton.js

```diff
@@ -24,9 +24,6 @@ const fontColor = color => `
 `
 
 const StyledLinkButton = styled(Link)`
-svg {
```

Comment:
Why remove this? I think it is there bc we pass an svg as a child to LinkButton in ScholarshipsCard (tho not in use atm)

Reply:
thats a wrong css declararion, inside there it was only a value but no css valid declararion.

if you tell me whats the correct css declaration we can add it again :)